spec/query_builder_spec.cr

```diff
@@ -55,6 +55,24 @@ describe "LuckyRecord::QueryBuilder" do
     query.statement.should eq "SELECT COUNT(*) FROM users"
     query.args.should eq [] of String
   end
+
+  describe "#select" do
+    it "can specify a column to be selected" do
```

Comment:
I think this spec can be removed since the one below will cover both cases automatically

Reply:
Yeah, it just moved me along tdd'ing this. Never afraid to remove specs 😈 ✅